README.md

```diff
@@ -25,6 +25,51 @@ For example, if you have an application named `hello-world` you can access it on
 The [discovery script](https://github.com/adobe-apiplatform/apigateway/blob/master/api-gateway-config/marathon-service-discovery.sh) is provided as an example for a quick-start and it can be replaced with your favourite discovery mechanism.
 The script updates a [configuration file](https://github.com/adobe-apiplatform/apigateway/blob/master/api-gateway-config/environment.conf.d/api-gateway-upstreams.http.conf) containing all the NGINX upstreams that are used in the [config file](https://github.com/adobe-apiplatform/apigateway/blob/master/api-gateway-config/conf.d/marathon_apis.conf#L36).
 
+#### Accessing a Marathon app remotely
+
+All you need to do is to create a DNS entry like `*.api.example.com` and have it resolve to the nodes running the Gateway.
+
+Assuming there is an application deployed in Marathon called `hello-world` it can be accessed at the URL: `http://hello-world.api.example.com`
+The Gateway is automatically proxying the request to the `hello-world` application in Marathon.
+
+If you call `http://my-custom-app.api.example.com` the Gateway will proxy to a Marathon app named `my-custom-app`.
+If the application is not deployed in Marathon a 5xx error is returned back to the client. This behaviour is ofcourse configurable.
+
+#### Running with a different configuration folder
+
+One way to simplify the configuration of the Gateway is to copy the `api-gateway-config` folder in the cloud, having all nodes syncing from that location.
```

Comment:
Perhaps cloud is a bit too generic, shouldn't it be an "object store" or something similar. Alternatively, if this currently only works with S3 it should be S3 and have a mention in the roadmap etc. that the intention is to support other services.